### DISCUSSION TITLE
浙江大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#7] [University]
> 回答来自群友，我仅负责搬运

zju 紫金港：
三种寝室，都是上床下桌不过布局不同，开奖；
有的寝有独立卫浴，有的寝是每层楼的公用卫浴，独立卫浴不断水，不掌握公用卫浴具体情况；
无早晚自习，跑步一学期 48 次每次男 3.5km 女 2.5km 限时均为 45 分钟，早八点前和下午三点后到晚十点半前可跑；可骑车；
允许外卖，宿舍区离外卖点近，地铁距学校各校门最远不超过一公里；
有独立卫浴的寝室可以租用单独的洗衣机，一年（两年？）四百，公用洗衣房一桶 5-7 元，烘干机一桶 12-24 元（时长不同），人工洗鞋 19 元/次；
校园网包月，套餐 10 元至 50 元不等。流量无限，10 元档限速 10Mbps，30/50 元档速度更快；无线网使用三个 SSID 广播，其中两个支持 IPv4 IPv6 双栈。注意：不开放 UDP 53 端口出网。
食堂凑合，偶发食品安全问题，价格一顿 8 元至 18 元不等。

关于 cs 课设：
https://telegra.ph/Untitled-06-24-11

**Comment by Shuenhoy** [comment]
紫金港是有一种上下铺+一排桌子独卫四人间的。

**Reply by ZJUGuoShuai** [reply]
发现曾经课程大作业队友

**Reply by Shuenhoy** [reply]
> 发现曾经课程大作业队友

听说您要在qh老师那里读研了？

**Reply by ZJUGuoShuai** [reply]
> > 发现曾经课程大作业队友
> 
> 听说您要在qh老师那里读研了？

咦，你怎么知道？不过qh那里不太适合我，换了导师。

**Reply by Shuenhoy** [reply]
> > > 发现曾经课程大作业队友
> > 
> > 
> > 听说您要在qh老师那里读研了？
> 
> 咦，你怎么知道？不过qh那里不太适合我，换了导师。

我听一个今年在qh那里保研的朋友提到的

**Comment by palemoons** [comment]
大一新生都会进紫金港校区，至少会在这里住两年。部分学院大三/四会搬去其他校区。这里只补充紫金港的情况。

1. 寝室可以自己商量购买洗衣机，需要向楼长申请（不难，基本上都会通过）
2. 寒暑假放的晚，开学也较晚（应该每年都可以评比全国前三x 大概是因为期中考试的考试周不算进教学时间内；据说下一学年会调整期中考试时间，能够提早放假
3. 最近的地铁口离宿舍区有1.5km的距离，目前开通了摆渡车服务。
4. 校园网偶尔会不稳定，大部分时候在宿舍可以正常打游戏
5. 断电不断网，每年国庆节后到第二年夏季之间23:30-6:00左右断电。
6. 通宵可以去宿舍楼外的北街（很近），消费大概是30-50/人
7. 不会限制带电脑，有实力的同学完全可以在寝室搞台主机
8. 校园卡方面，目前因为和钉钉联动所以完全可以在钉钉上用校园宝余额支付，支付宝上充钱，平时不带卡也没关系，校门、宿舍门都可以刷脸通过
9. 银行卡学校会统一让大家办理中行借记卡，奖学金、勤工助学金都会在上面发放
10. 超市的话教育超市几乎每个宿舍区都有一家，有全家但是晚上12点之后就会关门。
11. 收快递统一在食堂旁边的邮局里，京东除外（在北街上有一个快递点）
12. 晚上不查寝、不锁门，大门锁了可以刷脸/刷卡走小门。
